tests/admin_api/test_initialization.py

```diff
@@ -59,3 +60,18 @@ async def test_private_token():
         private_token.oauth_url
         == f'https://{test_information.store_name}.myshopify.com/admin/oauth/access_token'
     )
+
+
+@pytest.mark.asyncio
+async def test_version_offline_token():
+    offline_token = await VersionOfflineToken.load(store_name=test_information.store_name)
+    print(offline_token.api_version)
+
+    assert offline_token.access_token == offline_token_data.access_token
+    assert offline_token.scope == offline_token_data.scope
+    assert offline_token.store_name == test_information.store_name
```

Comment:
I'm not sure what you are testing other than the test code you wrote in `tests/token_classes` ...  :thinking:

Reply:
Copy and paste issue :sweat_smile: